### PR TITLE
update images

### PR DESCRIPTION
**Pull request by masa213f**
Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: pause/Dockerfile [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Stage1: build from source
 FROM quay.io/cybozu/ubuntu-dev:20.04 AS build
 
-ARG K8S_VERSION=1.23.0
-ARG PAUSE_VERSION=3.6
+ARG K8S_VERSION=1.24.3
+ARG PAUSE_VERSION=3.7
```
[the remark this answers is not in the document]
Comment (masa213f) [reply pause/Dockerfile]: I see. I will update `K8S_VERSION` only.

The `pause` is scratch-based image and the source code has not been changed.
So I think we need not rebuild it. What do you think about it?

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

File: etcd/TAG [hunk 2]
```diff
@@ -1 +1 @@
-3.5.4.1
```
Comment (kmdkuk) [inline etcd/TAG:1]: I didn't think it was necessary to update the tag if it was just a README modification.

Comment (masa213f) [reply etcd/TAG:1]: The etcd image is a ubuntu-based image. So I intent to update the base image.

**Review by kmdkuk — APPROVED** [review]
LGTM

**Pull request merged**